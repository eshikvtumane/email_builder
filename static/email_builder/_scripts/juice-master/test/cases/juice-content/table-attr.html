<html>
<head>
    <style>
        p, td, img {
            text-align: center;
            vertical-align: middle;
        }

        table {
            background-image: url('test.png');
            background-color: red;
        }
    </style>
</head>
<body>
    <p>none</p>
    <table>
        <tr>
            <td>
                wide
            </td>
        </tr>
    </table>
</body>
</html>
