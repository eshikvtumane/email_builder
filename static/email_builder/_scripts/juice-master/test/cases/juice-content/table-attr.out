<html>
<head>
    
</head>
<body>
    <p style="text-align: center; vertical-align: middle;">none</p>
    <table style="background-image: url('test.png'); background-color: red;" background="url('test.png')" bgcolor="red">
        <tr>
            <td style="text-align: center; vertical-align: middle;" align="center" valign="middle">
                wide
            </td>
        </tr>
    </table>
</body>
</html>
